chapter_computational-performance/hybridize_vn.md

```diff
@@ -155,7 +155,12 @@ When designing Gluon, developers considered whether it would be possible to comb
 This led to a hybrid model that lets users develop and debug using pure imperative programming, while having the ability to convert most programs into symbolic programs to be run when product-level computing performance and deployment are required.
 -->
 
-*dịch đoạn phía trên*
+Trong quá khứ, hầu hết các framework đều chọn một trong hai phương án tiếp cận là lập trình mệnh lệnh và lập trình ký hiệu.
+Ví dụ, Theano, TensorFlow, Keras và CNTK đều xây dựng mô hình dạng ký hiệu.
+Ngược lại, Chainer và PyTorch tiếp cận theo hướng lập trình mệnh lệnh.
+Mô hình kiểu mệnh lệnh đã được tích hợp vào TensorFlow 2.0 (thông qua chế độ Eager) và Keras trong những bản cập nhật mới nhất.
+Khi thiết kế Gluon, các nhà phát triển đã cân nhắc liệu rằng có thể kết hợp ưu điểm của cả hai mô hình lập trình lại với nhau hay không.
+Nếu được một mô hình lai kiểu này sẽ giúp người dùng phát triển và gỡ lỗi bằng lập trình mệnh lệnh thuần, trong khi vẫn có khả năng chuyển đổi hầu như toàn bộ chương trình sang kiểu ký hiệu nếu yêu cầu hiệu năng tính toán cao và triển khai sản phẩm.
```

Comment:
```suggestion
Có được một mô hình tích hợp sẽ giúp người dùng phát triển và gỡ lỗi bằng lập trình mệnh lệnh thuần, đồng thời mang lại khả năng chuyển đổi hầu như toàn bộ chương trình sang dạng ký hiệu để chạy khi có yêu cầu triển khai với chất lượng tính toán cao cho sản phẩm. <!--hoặc `cấp độ sản phẩm`-->
```

Reply:
Từ `performance` trong Glossary đã thảo luận và lựa chọn sử dụng là từ `chất lượng` bạn nha.

Reply:
Câu dưới có vẻ hợp lý hơn rồi ạ.
Từ `performance` trong `computing performance` có vẻ thiên về tốc độ tính toán hơn? Không biết dịch là `chất lượng` có ổn không. Mọi người nghĩ sao ạ.

Reply:
Từ này một thời gian cả nhóm tranh cãi khi dịch cuốn `Machine Learning Yearning`, bạn có thể xem qua cuộc thảo luận *[tại đây](https://github.com/aivivn/Machine-Learning-Yearning-Vietnamese-Translation/pull/259)* nhé. Mình giải thích lại ở đây thì dài dòng.